src/models/TestResultsDAO.ts

```diff
@@ -126,7 +126,8 @@ export class TestResultsDAO {
     };
   }
 
-  public getTestCodesAndClassificationFromTestTypes(testTypeId: string, vehicleType: any, vehicleSize: any, vehicleConfiguration: any, noOfAxles: any) {
+  public getTestCodesAndClassificationFromTestTypes(testTypeId: string, vehicleType: any, vehicleSize: any, vehicleConfiguration: any, noOfAxles: any,
+                                                    euVehicleCategory: any, vehicleClass: any, vehicleSubclass: any, numberOfWheelsDriven: any) {
```

Comment:
surprised the linter didn't complain about this

Reply:
I think it's just at the limit